cilium/CFP-30572-non-default-deny-policies.md

```diff
@@ -0,0 +1,202 @@
+# CFP-30572: Non-default-Deny Network Policy
+
+**SIG: SIG-Policy**
+
+**Begin Design Discussion:** 2024-02-02
+
+**Cilium Release:** 1.16
+
+**Authors:** Casey Callendrello <cdc@isovalent.com>
+
+## Summary
+
+Allow users to create network policies that do not implicitly set endpoints in to a default-deny mode. 
+
+## Motivation
+
+Consider a large, multitenant cluster. The cluster administrator is distinct from
+the workload owners, and the workloads are diverse and pre-existing.
+
+The administrator may wish to create a Cilium Clusterwide Network Policy (CCNP)
```

Comment:
Very nit: 
```suggestion
The administrator may wish to create a Clusterwide Cilium Network Policy (CCNP)
```

Reply:
Cilium comes first in the acronym in the vast majority of matches in the codebase, I think the wording is correct as-is.

```
$ git grep ClusterwideCi | wc -l
2
$ git grep CiliumClusterwide | wc -l
500
```

Reply:
Oh, you're right, I didn't check before posting and I've always thought it was Clusterwide first and then the CNP part after, but I guess I had the reverse in my head all this time :)